rad/src/reducers/mod.rs

```diff
@@ -51,6 +57,10 @@ pub fn reduce(input: &RadonArray, reducer_code: RadonReducers) -> Result<RadonTy
             }
             RadonReducers::Mode => mode::mode(input),
             RadonReducers::DeviationStandard => deviation::standard(input),
+            RadonReducers::AverageMedian => match &context.active_wips {
+                Some(active_wips) if active_wips.wip0017() => median::median(input),
+                _ => error(),
+            },
```

Comment:
Here if `context.active_wips` is `None`, the WIP is assumed to be not activated. This is the default for external tools that use the radon library, because they may not have access to a witnet node so they don't know the active wips. However we need to ensure that the `None` case means "enabled" after this WIP is activated.

Reply:
What do you think about changing the `None` case to mean enabled even if the WIP is not activated yet? This way external tools can create and run requests that use the new operators, but if they deploy the request before the WIP is active they will get an `UnsupportedReducer` error.

Reply:
But if they create a request, they try it with this feature and obtain a proper result, but when send to mainnet, it fails... It wasnt a good test to try before...